### DISCUSSION TITLE
Handling warning during page transformation

### DISCUSSION THREAD

**Discussion by mohammadamer** [pnp/pnpframework#211] [Q&A]
Hey!

I've been working with page transformations and I got few warnings

- Asset was not transferred. Asset: /sites/abc/xyz/PublishingImages/820x400/Bygg_din_MAC_820x400.png?RenditionID=6
- Skipped copying field: Tag Source or target multi-value setting mis-match
- Could not retrieve field value from mapping, the contents were empty

How to handle the above warnings?
Appreciate your help

**Comment by jansenbe** [comment]
@mohammadamer : please create an issue with more details for the first bullet. For the second point: source and target fields need to match if you want to copy metadata, might be something wrong here. Also note that you need to restart your PS session when you update the page layout mapping or web part mapping files as they're cached after first use
